arc/portal_gallery_package/Manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+   "$schema": "https://gallery.azure.com/schemas/2015-04-01/manifest.json#",
+   "name": "AzureKeyVaultSecretsProvider",
+   "publisher": "Microsoft",
+   "version": "1.0.0-preview",
+   "displayName": "Azure Key Vault Secrets Provider (preview)",
+   "publisherDisplayName": "Microsoft",
+   "publisherLegalName": "Microsoft",
+   "summary": "Azure Key Vault Secrets Provider provides integration of Azure Key Vault with a Kubernetes cluster",
+   "longSummary": "Azure Key Vault Secrets Provider for Secrets Store CSI Driver allows you to get secret contents stored in an Azure Key Vault instance and use the Secrets Store CSI driver interface to mount them into Kubernetes pods.",
+   "description": "Azure Key Vault Secrets Provider arc extension for Secrets Store CSI Driver allows you to get secret contents stored in an Azure Key Vault instance and use the Secrets Store CSI driver interface to mount them into Kubernetes pods. Some of the features of this extension includes mounting secrets keys and certs to pod using a CSI Inline volume, supports pod portability with the SecretProviderClass CRD, support for Linux and Windows containers, supports sync with Kubernetes Secrets, supports auto rotation of secrets.",
```

Comment:
Is the extension supported on windows? I don't think we set `windows.enabled=true` so we might want to remove windows from description.

Reply:
removed